cirrus/server/tests/test_feature_manifest.py

```diff
@@ -1,23 +1,185 @@
+import json
 import unittest
 
+from fml_sdk import FmlError
+from parameterized import parameterized
+
 from cirrus.feature_manifest import FeatureManifestLanguage
+from cirrus.sdk import SDK
 
 
 class FeatureManifestLanguageTestCase(unittest.TestCase):
-    def test_compute_feature_configurations(self):
-        fml = FeatureManifestLanguage()
+    def setUp(self):
+        self.fml = FeatureManifestLanguage()
+        self.sdk = SDK()
+
+    def test_compute_feature_configurations_always_return_default_config(self):
+        enrolled_partial_configuration = {
+            "enrolledFeatureConfigMap": {},
+            "enrollments": [],
+            "events": [],
+        }
+
+        result = self.fml.compute_feature_configurations(enrolled_partial_configuration)
+
+        self.assertEqual(
+            result, {"example-feature": {"enabled": False, "something": "wicked"}}
+        )
 
-        enrolled_partial_configuration = [
-            {"id": 1, "name": "test1", "value": "true"},
-            {"id": 2, "name": "test2", "value": "false"},
+    @parameterized.expand(
+        [
+            [
+                "imported-module-1-included-feature-1",
+                False,
+            ],
+            ["example-feature", True],
+            ["", False],
         ]
-        feature_configurations = [
-            {"id": 1, "name": "feature1", "value": "true"},
-            {"id": 2, "name": "feature2", "value": "false"},
+    )
+    def test_compute_feature_configurations_feature(
+        self, feature_name, expected_feature_status
+    ):
+        enrolled_partial_configuration = {
+            "enrolledFeatureConfigMap": {
+                feature_name: {
+                    "branch": "treatment",
+                    "feature": {
+                        "featureId": feature_name,
+                        "value": {"enabled": True},
+                    },
+                    "featureId": feature_name,
+                    "slug": "experiment_slug",
+                }
+            },
+            "enrollments": [
+                {
+                    "slug": "experiment_slug",
+                    "status": {
+                        "Enrolled": {
+                            "branch": "treatment",
+                            "enrollment_id": "enrollment_id",
+                            "reason": "Qualified",
+                        }
+                    },
+                }
+            ],
+            "events": [
+                {
+                    "branch_slug": "treatment",
+                    "change": "Enrollment",
+                    "enrollment_id": "enrollment_id",
+                    "experiment_slug": "experiment_slug",
+                    "reason": None,
+                }
+            ],
+        }
```

Comment:
We could use [factories](https://factoryboy.readthedocs.io/en/stable/) to generate these so we don't need to worry about their structure and values in every test.